pkg/config/config.go

```diff
@@ -112,16 +114,27 @@ type SecurityCapabilityLists struct {
 
 // ParseFile parses config from a file.
 func ParseFile(path string) (Configuration, error) {
-	configBox := packr.New("Config", "../../examples")
-	var rawBytes []byte
 	var err error
+	var rawBytes []byte
+	configBox := packr.New("Config", "../../examples")
 	if path == "" {
 		rawBytes, err = configBox.Find("config.yaml")
+		if err != nil {
```

Comment:
Once you move the `if err != nil` below, this check can go away, since it'll be handled outside the `if/else`.